tsinfer/formats.py

```diff
@@ -2292,56 +2292,67 @@ def populations(self):
             yield Population(j, metadata=metadata)
 
 
-class SgkitSampleData(SampleData):
+class VariantData(SampleData):
 
     FORMAT_NAME = "tsinfer-sgkit-sample-data"
     FORMAT_VERSION = (0, 1)
 
-    def __init__(self, path, sgkit_samples_mask_name=None, sites_mask_name=None):
+    def __init__(
+        self,
+        path,
+        ancestral_allele_name_or_array,
```

Comment:
The ``_name_or_array`` suffix is redundant for all these isn't it? We don't put type information into most things, and since these all share the same suffix I don't see what problem it solves.

Reply:
Good point - fixed